### PR TITLE
Adding references

### PR DESCRIPTION
**Pull request by hagai-reuveni**
Adding project references instead of paths.

### CODE REVIEW

File: app/src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { FC } from 'react';
 import { render } from 'react-dom';
 import { HashRouter as Router } from 'react-router-dom';
-import { Header } from '@ts-monorepo/components/Header';
+import { Header } from '@ts-monorepo/components/src/Header';
```
Comment (sharonamr) [inline app/src/App.tsx]: we shouldn't have **src** in the import

File: app/tsconfig.json [hunk 2]
```diff
@@ -2,21 +2,10 @@
   "extends": "../base.tsconfig.json",
   "include": [
     "src",
+    "./webpack.config.js"
```
Comment (sharonamr) [inline app/tsconfig.json]: Why do we need it?

**Review by sharonamr — CHANGES_REQUESTED** [review]

File: app/tsconfig.json (hunk 2 above)
Comment (hagai-reuveni) [reply app/tsconfig.json]: It's not mandatory, but I think it's better to have it monitor by ts so we can have monitored

File: app/src/App.tsx (hunk 1 above)
Comment (hagai-reuveni) [reply app/src/App.tsx]: We can import from `@ts-monorepo/components`. This is what I meant when I said we can have it more elegant.

File: app/tsconfig.json (hunk 2 above)
Comment (hagai-reuveni) [reply app/tsconfig.json]: Removed it for now

File: app/src/App.tsx (hunk 1 above)
Comment (hagai-reuveni) [reply app/src/App.tsx]: Changed it to `import { Header } from '@ts-monorepo/components`

File: app/webpack.config.js [hunk 3]
```diff
@@ -15,7 +17,7 @@ module.exports = {
   resolve: {
     extensions: ['.tsx', '.ts', '.js'],
     alias: {
```
Comment (sharonamr) [inline app/webpack.config.js]: please remove this :-)

**Review by sharonamr — APPROVED** [review]

**Pull request merged**